packages/azure-http-specs/specs/azure/resource-manager/operation-templates/lro.tsp

```diff
@@ -0,0 +1,203 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.OperationTemplates;
+
+@resource("lroResources")
+model LroResource is TrackedResource<LroResourceProperties> {
+  ...ResourceNameParameter<LroResource>;
+}
+
+model LroResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+}
+
+@armResourceOperations
+interface Lro {
+  @scenario
+  @scenarioDoc("""
+    Resource PUT operation.
+    Service returns "Azure-AsyncOperation" on initial request.
+    final-state-via: Azure-AsyncOperation
+    
+    Expected verb: PUT
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "location": "eastus",
+      "properties": {
+        "description": "valid"
+      }
+    }
+    ```
+    
+    Expected status code: 201
+    Expected response header: Azure-AsyncOperation={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/operations/create_or_replace
+    
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro",
+      "name": "top",
+      "type": "topLevel",
+      "location": "eastus",
+      "properties": {
+        "description": "valid",
+        "provisioningState": "Succeeded"
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
```

Comment:
is this a correct `Azure-AsyncOperation` resource format? refer https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format

Reply:
This is the initial PUT response body, not `Azure-AsyncOperation` poll response.

For `Azure-AsyncOperation` resource, see https://github.com/Azure/typespec-azure/pull/1821/files/b833042b3de9d2fc450138ac956b51dc10eefd11#diff-acc7bdbf75c64c622679dfccf6922b83b1512d07a5ce265894d681140d53e4daR72-R83

```json
{
  "id": "/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao",
  "name": "aao",
  "status" : "InProgress"
}
```